ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/userManagementServiceAgent.service.ts

```diff
@@ -7,6 +7,6 @@ import { SessionServiceAgent } from './sessionServiceAgent.service';
 @Injectable()
 export class UserManagementServiceAgent extends SessionServiceAgent<void> {
   public deleteUser(userGuid: string): Observable<void> {
-    return this.deleteObservable(`/user/${userGuid}`).pipe(take(1));
+    return this.deleteObservable(`/user/${userGuid}`, null, true).pipe(take(1));
   }
```

Comment:
The `true` flag added here ensures the implementation of `deleteObservable` throws an error that we can catch with the catchError operator in the component.